add containers release note (#4902)
theforeman/foreman-documentation: guides/doc-Release_Notes/topics/foreman.adoc

//If there are no headline features, uncomment the following line:
//There are no highlights with Foreman {ProjectVersion}.

Containerized deployments::

> It is now possible to deploy using containers instead of classical distribution packages.

Reviewer: Just a suggestion for improvement with more focus on reader, but either works for upstream.

```suggestion
You can now deploy using containers instead of classical distribution packages.
```

Agama installer support::
Added support for the Agama installer (SLES16), expanding {Project}'s provisioning capabilities for SUSE Linux Enterprise systems.

Containerized deployments::
It is now possible to deploy using containers instead of classical distribution packages.

> See the {BaseURL}Quickstart/index-containerized-katello.html[Quick start guide] and {BaseURL}Installing_Server/index-containerized-katello.html[Installing {ProjectServerTitle} guide] for details.

Author: This intentionally links to the Katello version of the docs, regardless of the build, as that's the only one having containers support right now.
